Statistics.cs

```diff
@@ -483,73 +494,84 @@ public void DisplayProvinceList()
             foreach (KeyValuePair<string, CityInfo> city in CityCatalogue)
             {
                 string[] nameParts = city.Key.Split('|');
                 if (nameParts[0].ToLower() == cityName.ToLower())
-                {
                     matchedCities.Add(city.Value);
-                }
             }
 
             if (matchedCities.Count == 0)
             {
-                Console.WriteLine($"No city exists named {cityName}.\n");
+                //No cities were found
+                Console.WriteLine($"No city exists named {CapitalizeString(cityName)}.\n");
                 return null;
             }
             else if (matchedCities.Count > 1)
             {
+                //Display the options for multiple cities under the same name
                 Console.WriteLine($"Multiple Cities Named {matchedCities[0].GetCityName()}. Please Select Desired City.\n");
 
                 for (int i = 0; i < matchedCities.Count; i++)
-                {
                     Console.WriteLine($"\t{i + 1}) {matchedCities[i].GetCityName()}, {matchedCities[i].GetProvince()}");
-                }
 
+                //Prompt for user input
                 do
                 {
                     Console.Write("\nPlease Select Desired City (ex 1, 2): ");
                     int choice;
                     if (int.TryParse(Console.ReadLine(), out choice) && choice > 0 && choice <= matchedCities.Count)
-                    {
                         return matchedCities[choice - 1];
-                    }
+
                     Console.WriteLine("Invalid choice. Please try again.");
                 } while (true);
 
             }
             else
-            {
                 return matchedCities[0];
-            }
         }
 
+        /// <summary>
+        /// Capitalize the first letter of a string
+        /// </summary>
+        /// <param name="toCapitalize">The string to capitalize</param>
+        /// <returns>The first letter capitalized</returns>
         private string CapitalizeString(string toCapitalize)
         {
             if (toCapitalize.Length == 0)
                 return toCapitalize;
             else if (toCapitalize.Length == 1)
                 return toCapitalize.ToUpper();
             else
-                return toCapitalize.Substring(0, 1).ToUpper() + toCapitalize.Substring(1);
+            {
+                if (toCapitalize.Split(' ').Length > 1)
+                {
+                    //Handle 1+ words
+                    string capitalized = "";
+                    foreach (string word in toCapitalize.Split(' '))
+                        capitalized += word.Substring(0, 1).ToUpper() + word.Substring(1).ToLower() + " ";
+
+                    return capitalized.Trim();
+                }
+                else
+                    return (toCapitalize.Substring(0, 1).ToUpper() + toCapitalize.Substring(1).ToLower()).Trim();
+            }
         }
```

Comment:
Nicely done, I don't think the final else is required though, if it's one word it should still handle it correctly. Going to merge so I can add some fixes too and this will be one